docs/_includes/date-picker.html

```diff
@@ -0,0 +1,144 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+    <p>Arrow keys should navigate between dates, but this is an implementation detail that is not yet included in skin.</p>
+
+    <h3>Single Pane</h3>
+
+    <div class="date-picker" aria-label="date picker">
+        <div class="date-picker__header">
+            <div class="date-picker__header--inner">
+                <button
+                    class="icon-btn icon-btn--small icon-btn--transparent"
+                    aria-label="Previous Month - July 2019"
+                    variant="icon"
+                >
+                    <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
+                        {% include symbol.html name="chevron-left" %}
+                    </svg>
+                </button>
+                <h3>August 2019</h3>
+                <button
+                    class="icon-btn icon-btn--small icon-btn--transparent"
+                    aria-label="Next Month - September 2019"
+                    variant="icon"
+                >
+                    <svg aria-hidden="true" class="icon icon--chevron-right" focusable="false">
+                        {% include symbol.html name="chevron-right" %}
+                    </svg>
+                </button>
+            </div>
+        </div>
+        <div class="date-picker__body">
+            <div class="date-picker__calendar-cell">
+                {% include calendar-month.html dynamic="true" %}
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="date-picker date-picker--single-month" aria-label="date picker">
```

Comment:
Is `.date-picker--single-month` modifier necessary on the base class? It should be the default and not needed, right?